core/modules/layout/config/layout.settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "_config_name": "layout.settings",
+    "templates": [
```

Comment:
not entirely sure, but should this perhaps be `layout_allowed_templates` or `layout_enabled_templates` or something? I randomly looked at https://github.com/backdrop/backdrop/blob/1.x/core/modules/book/config/book.settings.json#L3 for example and we use `book_allowed_types` there.